tests/framework/common.go

```diff
@@ -1386,5 +1387,32 @@ func (td *OsmTestData) GrabLogs() error {
 		}
 	}
 
+	// TODO: Migrate these config collections to CLI
+	pods, err := td.Client.CoreV1().Pods("").List(context.Background(), metav1.ListOptions{
+		// Reliable way to select injected pods
+		LabelSelector: "osm-proxy-uuid",
+	})
+
+	if err != nil {
+		return err
+	}
+
+	for _, pod := range pods.Items {
+		// injected pod, fetch envoy's config
+		podEnvoyConfigFilepath := td.GetTestFilePath(fmt.Sprintf("envoy_%s_%s.conf", pod.Namespace, pod.Name))
+
+		stdout, stderr, err := td.RunLocal("../../bin/osm", []string{"proxy", "dump-config", pod.Name, "--namespace", pod.Namespace})
+		if err != nil {
+			td.T.Logf("error running dump-config script")
+			td.T.Logf("stdout:\n%s", stdout)
+			td.T.Logf("stderr:\n%s", stderr)
+		} else {
```

Comment:
If we don't have an actual test for this CLI command, it seems like these errors may go unnoticed given getting to this point means there's been some other failure and Ginkgo makes those very obvious.

Reply:
we are going to work on tooling next few days if that's the concern. 
The immediate reason for these are the intermittent CI failures, this would allow us extended visibility on the pipes